### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by ztalha6**

### CODE REVIEW

File: lib/data/path_data.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const String PATH_ACTIVITY = 'android/app/src/main/';
+
+const String PATH_APP_LEVEL_GRADLE = 'android/app/build.gradle';
+const String PATH_TO_PLIST = 'ios/Runner/Info.plist';
+const String PATH_TO_PODFILE = 'ios/Podfile';
+const String PATH_AD_UNIT_ID = 'lib/ad_unit_ids/ad_unit_id.dart';
+const String PATH_MAIN = 'lib/main.dart';
+const String PATH_GRADLE_PROPERTIES = 'android/gradle.properties';
+
+// const String APPLICATION_ID = """\n        <meta-data
```
Comment (ztalha6) [inline lib/data/path_data.dart]: @waleed-sofstica Please mention the usage of these variables. For now, it is not been used in any file. Are these useless?

Comment (Syed-Waleed-Shah) [reply lib/data/path_data.dart]: All of these variables are paths of files which are being configured. Each if these paths are being used

Comment (Syed-Waleed-Shah) [reply lib/data/path_data.dart]: The gradle.properties variable will be created while configuration if it doesn't already exists.
The main.dart file is used to place the initialization code of ads

Comment (talhaz6) [reply lib/data/path_data.dart:15]: @waleed-sofstica the variable I commented is not being use anywhere. Give me a heads up to remove comments

Comment (ztalha6) [reply lib/data/path_data.dart]: @Syed-Waleed-Shah The variable I commented is not being use anywhere. Give me e a heads up to remove comments

File: lib/global/data/path_data.dart [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const String PATH_MANIFEST = 'android/app/src/main/AndroidManifest.xml';
```
Comment (Syed-Waleed-Shah) [inline lib/global/data/path_data.dart]: rename this file to file_paths.dart

File: lib/global/data/path_data.dart [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const String PATH_TO_PODFILE = 'ios/Podfile';
+const String PATH_AD_UNIT_ID = 'lib/ad_unit_ids/ad_unit_id.dart';
+const String PATH_MAIN = 'lib/main.dart';
+const String PATH_GRADLE_PROPERTIES = 'android/gradle.properties';
+
+const String STRING_PODFILE_IMPORT_GOOGLE_ADS =
+    """pod 'Google-Mobile-Ads-SDK'""";
+const String STRING_PODFILE_IMPORT_APPLOVIN =
+    """pod 'GoogleMobileAdsMediationAppLovin'""";
+
```
Comment (Syed-Waleed-Shah) [inline lib/global/data/path_data.dart]: move these constants to sdk_imports.dart (create new file)

Comment (Syed-Waleed-Shah) [inline lib/global/data/path_data.dart]: move these constants to sdk_imports.dart (create new file)

**Review by Syed-Waleed-Shah — CHANGES_REQUESTED** [review]

File: lib/global/data/path_data.dart (hunk 3 above)
Comment (ztalha6) [reply lib/global/data/path_data.dart]: these must be in the code string file as these are code lines that are being written.

**Pull request merged**